src/CreatePackage.php

```diff
@@ -0,0 +1,226 @@
+use Symfony\Component\Console\Question\ConfirmationQuestion;
+use Symfony\Component\Console\Question\Question;
+
+class CreatePackage extends Command
+{
+    protected function configure()
+    {
+        $this
+            ->setName('create-package')
+            ->setDescription('Creates a new package')
+            ->setHelp(
+                'The new package will be created in the directory of provided package name.'
+                . ' The package will be initialized with the whole basic structure.'
+                . ' It will be possible to choose namespace, based on provided package name,'
+                . ' or provide another namespace. At the end of the process it is possible to'
+                . ' create initial commit with the whole package structure.'
+                . PHP_EOL . PHP_EOL
+                . 'Package is created with documentation templates, all support files,'
+                . ' QA tools (CodeSniffer and PHPUnit), default ConfigProvider with tests,'
+                . ' Travis CI and coveralls configuration.'
+                . PHP_EOL . PHP_EOL
+                . 'Currently it supports zendframework and zfcampus organizations packages:'
+                . ' these must start with "zend-" or "zf-" respectively.'
+            )
+            ->addArgument('name', InputArgument::REQUIRED, 'The name of the package to create');
+    }
+
+    protected function initialize(InputInterface $input, OutputInterface $output)
+    {
+        parent::initialize($input, $output);
+
+        $name = $input->getArgument('name');
+        if (! preg_match('/^(zend|zf)-[a-z0-9-]+$/', $name)) {
+            throw new InvalidArgumentException(
+                'Invalid package name, must be prefixed with "zend-" or "zf-"'
+                . ' and can contain only lowercase letters, numbers and dash.'
+            );
+        }
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $repo = $input->getArgument('name');
+
+        $errOutput = $output instanceof ConsoleOutputInterface ? $output->getErrorOutput() : $output;
+
+        if (is_dir($repo)) {
+            $errOutput->writeln(sprintf(
+                '<error>Directory with name "%s" already exists</error>',
+                $repo
+            ));
+
+            return;
+        }
+
+        if (! mkdir($repo, 0775) && ! is_dir($repo)) {
+            $errOutput->writeln(sprintf(
+                '<error>Cannot create package directory "%s"</error>',
+                $repo
+            ));
+
+            return;
+        }
+
+        $namespace = $this->determineNamespace($input, $output, $repo);
+
+        $replacement = [
+            '{org}' => $this->getOrg($repo),
+            '{repo}' => $repo,
+            '{category}' => $this->getCategory($repo),
+            '{namespace}' => $namespace,
+            '{namespace-test}' => preg_replace('/^(Zend|ZF)/', '\\1Test', $namespace),
```

Comment:
This would need to change as well. I'd write a function that does the following:

```php
$parts = explode('\\', $namespace);
$first = array_shift($parts);
return $first . 'Test';
```